setup.py

```diff
@@ -28,6 +28,7 @@
     description="A machine learning toolkit dedicated to time-series data",
     long_description=README,
     long_description_content_type='text/markdown',
+    license='2-clause BSD License',
```

Comment:
This would be the [correct SPDX identifier](https://spdx.org/licenses/BSD-2-Clause.html):

```suggestion
    license='BSD 2-Clause "Simplified" License',
```

Reply:
SPDX and OSI apparently disagree on the long name for this license (see https://opensource.org/licenses/BSD-2-Clause and https://spdx.org/licenses/BSD-2-Clause.html). I've switched to the short identifier which they do agree on.

Reply:
Looks good 👍